webapp/Makefile

```diff
@@ -57,6 +57,7 @@ dynacred_npm: cothority_npm
 		( echo "Moving changes to dynacred"; cp -a src/lib/dynacred/ ../dynacred/src ); \
 	rm -rf src/lib/dynacred; \
 	find src/ -name "*.ts" | xargs perl -pi -e "s:src/lib/dynacred:\@c4dt/dynacred:"
+	cd ../dynacred; npm run lint:fix
```

Comment:
Hum, the perl changes shouldn't yield any issue with linting, why adding it?

Reply:
yes, linting changes, because the order of the imports is not correct anymore with respect to the new paths...